packages/common/storage/src/typed/storages/NumberTypedStorage.ts

```diff
@@ -2,13 +2,13 @@
 import { TypedStorage } from './TypedStorage';
 
 export class NumberTypedStorage extends TypedStorage<number> {
-  public increase(): void {
+  public increase(offset = 1): void {
     const value = this.get() ?? 0;
-    this.set(value + 1);
+    this.set(value + offset);
   }
 
-  public decrease(): void {
+  public decrease(offset = 1): void {
     const value = this.get() ?? 0;
-    this.set(value - 1);
+    this.set(value - offset);
```

Comment:
I hope this will be added to a new method `add` and `subtract`.

Reply:
It might be a better solution to break it down into several functions instead of just implementing in just one function(`increase`) it's a good solution.

However, in the case of the `Increase` function, I think It can be better get offset as a parameter. The reason why I think is as follows.

1. We don't know that the function name increase itself increases by 1(only just 1). who can think increase 10 or 100

2. Since the name `add` is to add two numbers, what and what do you add when you call add just one parameter like `add(10)`? This can lead to confusion.

```tsx

// when make add function
const INCREASE_OFFSET = 5;

const storage = createTypedLocalStorage('key', {
  initialValue: 0,
});

storage.add(INCREASE_OFFSET);

// when make increase function to accept parameter
const INCREASE_OFFSET = 5;

const storage = createTypedLocalStorage('key', {
  initialValue: 0,
});

storage.increase(INCREASE_OFFSET);

```

Looking at the code, I think both are okay, so I'd like to hear from other people.

Reply:
Yes I have thought about it again, your changes look great.